packages/slate-editor/src/modules/editor-v4-galleries/lib/normalizeInvalidGallery.ts

```diff
@@ -1,21 +1,18 @@
-import { Editor, Element, Node, NodeEntry, Transforms } from 'slate';
+import { EditorCommands } from '@prezly/slate-commons';
+import { GALLERY_NODE_TYPE, GalleryNode, isGalleryNode } from '@prezly/slate-types';
+import { Editor, Node, NodeEntry, Transforms } from 'slate';
 
-import { GALLERY_TYPE } from '../constants';
-import { GalleryElementType } from '../types';
-
-import isGalleryElement from './isGalleryElement';
-
-// We cannot use `isGalleryElement` here, because it checks for `images` array length.
-const isGalleryElementType = (node: any): node is GalleryElementType =>
-    Element.isElement(node) && node.type === GALLERY_TYPE;
+// We cannot use `isGalleryNode` here, because it checks for `images` array length.
+const isGalleryNodeType = (node: any): node is GalleryNode =>
+    EditorCommands.isElementWithType(node) && node.type === GALLERY_NODE_TYPE;
```

Comment:
Node for self. similar-but-different type guards: `isGalleryNode` vs `isGalleryNodeType`.